src/platform/datapath_xplat.c

```diff
@@ -84,24 +73,26 @@ CxPlatDataPathUninitialize(
 
 _IRQL_requires_max_(PASSIVE_LEVEL)
 void
-CxPlatDataPathUpdateConfig(
+CxPlatDataPathUpdatePollingIdleTimeout(
     _In_ CXPLAT_DATAPATH* Datapath,
-    _In_ QUIC_GLOBAL_EXECUTION_CONFIG* Config
+    _In_ uint32_t PollingIdleTimeoutUs
     )
 {
-    DataPathUpdateConfig(Datapath, Config);
+    DataPathUpdatePollingIdleTimeout(Datapath, PollingIdleTimeoutUs);
     if (Datapath->RawDataPath) {
-        RawDataPathUpdateConfig(Datapath->RawDataPath, Config);
+        RawDataPathUpdatePollingIdleTimeout(
+            Datapath->RawDataPath, PollingIdleTimeoutUs);
     }
 }
 
 _IRQL_requires_max_(DISPATCH_LEVEL)
-uint32_t
+CXPLAT_DATAPATH_FEATURES
 CxPlatDataPathGetSupportedFeatures(
-    _In_ CXPLAT_DATAPATH* Datapath
+    _In_ CXPLAT_DATAPATH* Datapath,
+    _In_ CXPLAT_SOCKET_FLAGS SocketFlags
```

Comment:
Is there a specific reason to use socket flags here instead of a "IncludeRawDatapath" boolean?
Most socket flags don't mean anything in this context, and callers seems to be crafting socket flags just for this.

Reply:
It allows us to return features based on the type of socket, such as TCP or PCP or XDP. Right now, it's just for XDP, but in the future we might want to adapt further.